core/src/main/java/tc/oc/pgm/timelimit/TimeLimitModule.java

```diff
@@ -83,14 +85,25 @@ public TimeLimitModule parse(MapFactory factory, Logger logger, Document doc)
       return oldTimeLimit;
     }
 
-    private static @Nullable TimeLimit parseTimeLimit(MapFactory factory, Element el)
+    private static @Nullable TimeLimit parseTimeLimit(MapFactory factory, Element el, String child)
         throws InvalidXMLException {
-      el = el.getChild("time");
+      el = el.getChild(child);
```

Comment:
Doesn't seem like something that would need changing, given "time" is the only param ever used?
Maybe just rename this function to `parseChildTimeLimit` and rename `el` to `parent`, to better reflect what it does